src/pageServices/contentScripts/highlight.js

```diff
@@ -287,24 +291,39 @@ export const highlightOnPage = () => {
     events.forEach((eventName) => {
       document.removeEventListener(eventName, scrollListenerCallback, true);
     });
-    document.removeEventListener("click", clickListener);
+
+    document.removeEventListener("dblclick", onElementDblClick);
+
     listenersAreSet = false;
   };
 
   const removeHighlight = (callback) => () => {
     removeEventListeners(removeHighlightElements(callback));
   };
 
-  const clickListener = (event) => {
-    if (!event.clientX && !event.clientY) return;
+  const onElementDblClick = (evt) => {
+    const elements = predictedElements?.filter((element) => element.jdnHash === evt.target.id);
+    const notAddedToPO = () => elements.some(({ generate }) => !generate);
+
+    sendMessage({
+      message: "ELEMENT_UNSET_ACTIVE",
+      param: elements[0].jdnHash,
+    });
+
+    if (notAddedToPO()) {
+      sendMessage({
+        message: "TOGGLE_ELEMENT",
+        param: elements.filter((element) => !element.generate),
+      });
+    }
   };
 
   const setDocumentListeners = () => {
     events.forEach((eventName) => {
       document.addEventListener(eventName, scrollListenerCallback, true);
     });
 
-    document.addEventListener("click", clickListener);
+    document.addEventListener("dblclick", (evt) => onElementDblClick(evt));
```

Comment:
I believe, it can be shorter: `document.addEventListener("dblclick", onElementDblClick);`